datahub-web-react/src/app/tags/TagsTable.tsx

```diff
@@ -119,12 +171,19 @@ const TagsTable = ({ searchQuery, searchData, loading: propLoading, networkStatu
                                 setEditingTag(record.entity.urn);
                                 setShowEdit(true);
                             }}
+                            onDelete={() => {
+                                if (canManageTags) {
+                                    deleteTag(record.entity.urn);
```

Comment:
Can we make it such that the delete button is disabled altogether if the user does not have permission to delete?

Reply:
I see - we need this for the modal display. Hmm maybe there is no other way that's easy